sormas-rest/pom.xml

```diff
@@ -128,87 +130,53 @@
 			</plugin>
 
 
-			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
-				<artifactId>properties-maven-plugin</artifactId>
-				<version>1.0.0</version>
-				<executions>
-					<execution>
-						<phase>initialize</phase>
-						<goals>
-							<goal>read-project-properties</goal>
-						</goals>
-						<configuration>
-							<files>
-								<file>src/main/resources/workflow-documentation.properties</file>
-							</files>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
-
 			<!-- Swagger Generation -->
 			<plugin>
-				<groupId>io.openapitools.swagger</groupId>
+				<groupId>io.swagger.core.v3</groupId>
 				<artifactId>swagger-maven-plugin</artifactId>
+				<version>2.2.3</version>
```

Comment:
please try if there is a way to inherit this from from base such that the `swagger.version` tag controls  everything.

Reply:
Done ([cf5b813](https://github.com/hzi-braunschweig/SORMAS-Project/pull/11202/commits/cf5b813abb3a973f6df3d1eb8229bd8cbeecb9a8))